repose-aggregator/components/filters/keystone-v2-filter/build.gradle

```diff
@@ -41,3 +41,16 @@ jaxb {
     }
 }
 
+jacocoTestCoverageVerification {
+    violationRules {
+        rules.forEach { rule ->
+            rule.limits.forEach { limit ->
+                if (limit.counter == 'INSTRUCTION' && limit.value == 'COVEREDRATIO') {
+                    // todo: set to the current coverage -- need more coverage, but we don't want the build to fail for the baseline
+                    limit.minimum = 0.46
```

Comment:
why is this one so low?

Reply:
I think it's almost all from missing generated source coverage. For this project, we configure XJC to add builder methods via the `-Xflient-api` flag:
```
jaxb {
    xjc {
        args = ["-Xdefault-value", "-Xfluent-api"]
    }
}
```

Well, we don't use all of the builder methods in our tests, so we don't cover those paths.

We may also be missing legitimate coverage of certain configurations, but I couldn't say without digging into the report more.

Reply:
le sigh